packages/exec/src/toolrunner.ts

```diff
@@ -524,6 +524,14 @@ export class ToolRunner extends events.EventEmitter {
           resolve(exitCode)
         }
       })
+
+      if (this.options.stdin) {
+        if (!cp.stdin) {
```

Comment:
is this a buffer or a stream?  https://nodejs.org/api/child_process.html#child_process_subprocess_stdin

Reply:
Responded to this above: https://github.com/actions/toolkit/pull/360#discussion_r406372534